spec/models/portfolio_item_spec.rb

```diff
@@ -35,4 +35,12 @@
       expect(item).to be_valid
     end
   end
+
+  context "default socpe" do
+    it "returns portfolio_items sorted by case insensitive names" do
+      %w[ab bb Bc Ad].each { |name| create(:portfolio_item, :name => name) }
+
+      expect(PortfolioItem.all.collect(&:name)).to eq(%w[ab Ad bb Bc])
```

Comment:
I know it's just a spec, but `PortfolioItem.pluck(:name)` will do this and generate `SELECT portfolio_items.name FROM portfolio_items;` so it'll run faster and not pull every field + the default scope. It doesn't need to change, just kind of a nice feature.

Reply:
Great point, I'll get that in.  Performance is always important.  👍

Reply:
I was going to leave the same comment, but thought I might see what SQL both commands generate.

```
2.5.5 :001 > Portfolio.pluck(:name)
   (10.9ms)  SELECT "portfolios"."name" FROM "portfolios" WHERE "portfolios"."discarded_at" IS NULL
 => ["third test", "tml", "Foo gg", "name foo", "JL-portfolio", "LGTest10", "DB Tower", "mslemr's Portfolio", "Madhu_test", "benny-test", "GM Test", "Test Minion Portfolio", "Pagination portfolio", "ShareTest1", "ShareTest2", "ITSM Products", "Docs", "JL Portfolio", "GM Test", "docs test 2", "magic unicorn", "Copy of GM Test", "ep-test", "Test100", "Copy of DB Tower", "LGTest2", "IQE Test", "Copy of Test Minion Portfolio", "LGTestNoTags", "Copy of benny-test", "LGTest11", "GM Test", "Benny test", "Copy of ShareTest2", "Delete test", "MMarosi", "MMarosi new orders", "Drew B test", "Copy of LGPortfolio", "LGPortfolio", "hhhhhh", "LG New Portfolio Test", "NEW1234", "New Endpoint Test", "test", "PMM TEST", "Madhu Test", "Test 88974", "testlg", "docs-test", "123"] 
2.5.5 :002 > Portfolio.all.collect(&:name)
  Portfolio Load (7.9ms)  SELECT "portfolios".* FROM "portfolios" WHERE "portfolios"."discarded_at" IS NULL
 => ["third test", "tml", "Foo gg", "name foo", "JL-portfolio", "LGTest10", "DB Tower", "mslemr's Portfolio", "Madhu_test", "benny-test", "GM Test", "Test Minion Portfolio", "Pagination portfolio", "ShareTest1", "ShareTest2", "ITSM Products", "Docs", "JL Portfolio", "GM Test", "docs test 2", "magic unicorn", "Copy of GM Test", "ep-test", "Test100", "Copy of DB Tower", "LGTest2", "IQE Test", "Copy of Test Minion Portfolio", "LGTestNoTags", "Copy of benny-test", "LGTest11", "GM Test", "Benny test", "Copy of ShareTest2", "Delete test", "MMarosi", "MMarosi new orders", "Drew B test", "Copy of LGPortfolio", "LGPortfolio", "hhhhhh", "LG New Portfolio Test", "NEW1234", "New Endpoint Test", "test", "PMM TEST", "Madhu Test", "Test 88974", "testlg", "docs-test", "123"] 
2.5.5 :003 > 
```

You can see the `Portfolio Load` on the collect version, but it actually runs faster than the `:pluck` variant. I assumed caching, so I exited console and came back in:

```
2.5.5 :001 > Portfolio.all.collect(&:name)
  Portfolio Load (7.6ms)  SELECT "portfolios".* FROM "portfolios" WHERE "portfolios"."discarded_at" IS NULL
 => ["third test", "tml", "Foo gg", "name foo", "JL-portfolio", "LGTest10", "DB Tower", "mslemr's Portfolio", "Madhu_test", "benny-test", "GM Test", "Test Minion Portfolio", "Pagination portfolio", "ShareTest1", "ShareTest2", "ITSM Products", "Docs", "JL Portfolio", "GM Test", "docs test 2", "magic unicorn", "Copy of GM Test", "ep-test", "Test100", "Copy of DB Tower", "LGTest2", "IQE Test", "Copy of Test Minion Portfolio", "LGTestNoTags", "Copy of benny-test", "LGTest11", "GM Test", "Benny test", "Copy of ShareTest2", "Delete test", "MMarosi", "MMarosi new orders", "Drew B test", "Copy of LGPortfolio", "LGPortfolio", "hhhhhh", "LG New Portfolio Test", "NEW1234", "New Endpoint Test", "test", "PMM TEST", "Madhu Test", "Test 88974", "testlg", "docs-test", "123"] 
2.5.5 :002 > 
``` 
Even faster...

I think they have done a lot of work on the AR versions of `:pluck` `:collect` over the years. My nit was to suggest using `map` over `collect` as that used to be recommended as well, but I can't find any concrete evidence for using one over the other.

All in all a good conversation to have either way.